Tests/eppo/ClientTests.swift

```diff
@@ -16,7 +16,7 @@ public class AssignmentLoggerSpy {
     var wasCalled = false
     var lastAssignment: Assignment?
     var logCount = 0
-
+    
```

Comment:
ran xcode "re-indent"

Reply:
🧹 🧹 🧹

Reply:
are the trailing spaces intentional?

Reply:
XCode added that - I would like to bake https://github.com/apple/swift-format into this repo.

<img width="1216" alt="Screenshot 2024-06-12 at 9 51 52 AM" src="https://github.com/Eppo-exp/eppo-ios-sdk/assets/57361/f61bb77c-f3b1-456c-86d1-6a0f0e05f967">

Reply:
ugh i'm sorry you have to use xcode 😢

Reply:
🤣 I'm using xcode for the debugger but mostly writing code in cursor to have the ai assist. surprisingly it is working pretty well together! both programs reload from disk often.